molecule/alternative/prepare.yml

```diff
@@ -3,14 +3,15 @@
   hosts: localhost
   gather_facts: false
   vars:
-    # Version seeds to be specified here as molecule doesn't have access to ansible_version at this stage
-    version: 2.14.0
+    # Version seeds to be specified here as molecule doesn't have access to prometheus_version at this stage.
+    # It needs to match playblook.yml
+    prometheus_version: 2.14.0
   tasks:
     - name: download prometheus binary to local folder
       become: false
       get_url:
-        url: "https://github.com/prometheus/prometheus/releases/download/v{{ version }}/prometheus-{{ version }}.linux-amd64.tar.gz"
-        dest: "/tmp/prometheus-{{ version }}.linux-amd64.tar.gz"
+        url: "https://github.com/prometheus/prometheus/releases/download/v{{ prometheus_version }}/prometheus-{{ prometheus_version }}.linux-amd64.tar.gz"
```

Comment:
This is on purpose not set to `prometheus_version` so it won't pick up the version from `defaults`.